### ISSUE TITLE
parsedmarc crash

### ISSUE THREAD

**Issue by Ooviov1r** [domainaware/parsedmarc#67]
/opt/parsedmarc/bin/parsedmarc -v                    
6.2.2

running
/opt/parsedmarc/bin/parsedmarc -c /etc/parsedmarc.ini
after a short while the application crashes with the following error

   DEBUG:__init__.py:999:Found 0 messages in IMAP folder INBOX
Traceback (most recent call last):
  File "/opt/parsedmarc/bin/parsedmarc", line 10, in <module>
    sys.exit(_main())
  File "/opt/parsedmarc/lib/python3.6/site-packages/parsedmarc/cli.py", line 510, in _main
    strip_attachment_payloads=sa)
  File "/opt/parsedmarc/lib/python3.6/site-packages/parsedmarc/__init__.py", line 1678, in watch_inbox
    if response[1] == b'RECENT' and response[0] > 0:
IndexError: tuple index out of range

**Comment by seanthegeek** [comment]
Please try the latest git commit and see if it resolves the IMAP issue. It also has more useful debugging output:

To install it, run:

```
pip install -U git+https://github.com/domainaware/parsedmarc.git
```

**Comment by seanthegeek** [comment]
Fixed in the 6.3.0 release.
